maat-court-data-api/src/test/java/gov/uk/courtdata/util/RepositoryUtil.java

```diff
@@ -0,0 +1,25 @@
+package gov.uk.courtdata.util;
+
+import lombok.experimental.UtilityClass;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Arrays;
+import java.util.function.Consumer;
+
+@UtilityClass
+public class RepositoryUtil {
+
+    private static final Consumer<JpaRepository<?, ?>> DELETE_ALL_AND_FLUSH = repository -> {
```

Comment:
Is the static keyword necessary here? From what I read the UtilityClass annotation makes all members static. :)

Reply:
Yes, agreed, good spot! I have removed the superfluous `static` keyword as it is already implied by `@UtilityClass` annotation. So the `DELETE_ALL_AND_FLUSH` variable now looks like the below.

```
    private final Consumer<JpaRepository<?, ?>> DELETE_ALL_AND_FLUSH = repository -> {
        repository.deleteAll();
        repository.flush();
    };
```